src/SigasiProjectCreator/DotF/DotFfileParser.py

```diff
@@ -9,12 +9,25 @@
 import os
 import sys
 import glob
+import re
 
 from SigasiProjectCreator.ArgsAndFileParser import ArgsAndFileParser
 from .parseFile import parse_dotf
 from ..convertDotFtoCsv import rebase_file
 
 
+def abspath(path):
+    s_path = str(path)
+    if s_path.startswith('\\') or s_path.startswith('/') or s_path[1] == ':' or s_path.startswith('$'):
+        # this is an absolute path in Linux or Windows
```

Comment:
Please use `os.path.isabs()` instead. What does the `$` mean?

Reply:
We want to be able to detect absolute Windows paths in Linux and vv. A.o. needed for unit tests.